schemas/viewdefinition/0.0.1-preview/ViewDefinition.json

```diff
@@ -0,0 +1,399 @@
+{
+    "id": "https://schema.management.azure.com/schemas/viewdefinition/0.0.1-preview/ViewDefinition.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "View Definition",
+    "description": "An Azure Managed Application View Definition.",
+    "type": "object",
+    "properties": {
+        "$schema": {
+            "type": "string",
+            "description": "JSON schema reference"
+        },
+        "contentVersion": {
+            "type": "string",
+            "pattern": "(^[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+$)",
+            "description": "A 4 number format for the version number of the view file. For example, 1.0.0.0"
+        },
+        "views": {
+            "type": "array",
+            "description": "The list of views.",
+            "items": {
+                "oneOf": [
+                    {
+                        "$ref": "#/definitions/Overview"
+                    },
+                    {
+                        "$ref": "#/definitions/Metrics"
+                    },
+                    {
+                        "$ref": "#/definitions/CustomResources"
+                    }
+                ]
+            }
+        }
+    },
+    "additionalProperties": false,
+    "required": [
+        "$schema",
+        "contentVersion",
+        "views"
+    ],
+    "definitions": {
+        "View": {
+            "discriminator": "kind",
+            "properties": {
+                "kind": {
+                    "type": "string",
+                    "description": "The kind of the view.",
+                    "x-ms-enum": {
+                        "name": "kind",
+                        "modelAsString": false
+                    },
+                    "enum": [
+                        "Overview",
+                        "Metrics",
+                        "CustomResources"
+                    ]
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the view"
```

Comment:
we should be consistent on the punctuation for the descriptions (either always add or never add) - I don't think we have a standard so would default to including it, given the use cases

Reply:
updated to add punctuation.